name.abuchen.portfolio/src/name/abuchen/portfolio/model/InvestmentPlan.java

```diff
@@ -49,6 +49,8 @@ public enum Type
 
     private List<Transaction> transactions = new ArrayList<>();
 
+    private LocalDate weeklyNominalDate;
```

Comment:
If we are adding a new property (which is persisted in XML) then we also have to persist it in the binary (protobuf) format.

However, I think we work without this date. Why not use the "DayOfWeek" to determine if the transaction has been moved (similarly how we use the DayOfMonth to see if the monthly transactions moved)